handlers/character.py

```diff
@@ -205,25 +207,33 @@ def get_status(other_username, db, chat_id, username):
     character = get_linked_character(db, chat_id, search_param)
 
     return (f'{character.name} | {character.race} {character._class} Level {character.level}\r\n'
-            f'HP: {character.current_hit_points}/{character.max_hit_points} | XP: {character.current_experience}')
+            f'HP: {character.current_hit_points}/{character.max_hit_points} | XP: {character.current_experience}/{character.experience_needed}')
+
+def get_currency(other_username, db, chat_id, username):
+    search_param = other_username if other_username != '' else username
+    search_param = utils.normalized_username(search_param)
+    character = get_linked_character(db, chat_id, search_param)
+
+    return (f'{character.name} currency pouch: ```\r\n'
```

Comment:
And this can be moved to the status command in one liner, like `Pouch: 0 pg, 135 gp, 0 sp, 0 cp, etc`